server/cmd/museum/main.go

```diff
@@ -682,7 +682,7 @@ func main() {
 	publicAPI.GET("/offers/black-friday", offerHandler.GetBlackFridayOffers)
 
 	setKnownAPIs(server.Routes())
-
+	lockController.ReleaseHostLock()
```

Comment:
We'll need to guarantee that none of the `NewFooController` methods above take any locks before we run this line. I don't think they do, and you've likely already checked, but just mentioning in case.

Reply:
Yes, I did check this part. May be it make sense to move it at the top [right after db migration], to reduce the chance of regressions. Does that make sense?

Reply:
Can do that too, yes, maybe that's safer.